core/import_data/import_project_comments.py

```diff
@@ -0,0 +1,35 @@
+import json
+import logging
+
+from django.conf import settings
+
+from core.import_data.utils import delete_old_data
+from core.import_data.utils import get_object_by_code
+from core.models import Project
+from core.models.project import ProjectComment
+
+
+logger = logging.getLogger(__name__)
+
+
+def import_project_comments():
+    file_path = settings.IMPORT_DATA_DIR / "progress_report" / "tbComment.json"
+    delete_old_data(ProjectComment, file_path)
+
+    with file_path.open("r") as f:
+        comments = json.load(f)
+
+    for index, item in enumerate(comments):
+        project = get_object_by_code(Project, item["Code"], "code", index)
+        if not project:
+            continue
```

Comment:
Hmmm I think it will be better if we log an warning here 🤔

Reply:
`get_object_by_code` already logs the missing project, so I omitted adding another one here. I followed the design pattern I noticed with other similar functions like `get_object_by_name`.